packages/babel-traverse/test/fixtures/parameters/function-declaration-in-params/exec.js

```diff
@@ -0,0 +1,11 @@
+if (parseInt(process.version.slice(1)) < 6) process.exit();
```

Comment:
Would be great if you can update to use the newly merged [`minNodeVersion` option](https://github.com/babel/babel/pull/5765) and get rid of the node check & eval here and a few other tests in this PR.